<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatDlg</class>
 <widget class="QDialog" name="ChatDlg">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 243, 243);</string>
  </property>
  <widget class="QPushButton" name="pushButton_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>一</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>×</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget_Main">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>371</width>
     <height>411</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>211</width>
       <height>106</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>注册/登录</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>211</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 30pt &quot;华文隶书&quot;;</string>
     </property>
     <property name="text">
      <string>Our Meeting</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QStackedWidget" name="stackedWidget_Register_Log">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>371</width>
       <height>401</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QPushButton" name="pushButton_log">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>280</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>登录</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;新宋体&quot;;</string>
       </property>
       <property name="text">
        <string>账号密码登录</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;新宋体&quot;;</string>
       </property>
       <property name="text">
        <string>账号</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>180</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;新宋体&quot;;</string>
       </property>
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_log_name">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>140</y>
         <width>271</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_log_pw">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>210</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_register_turn">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>350</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>10</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 75 16pt &quot;新宋体&quot;;</string>
       </property>
       <property name="text">
        <string>账号注册</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;新宋体&quot;;</string>
       </property>
       <property name="text">
        <string>账号</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;新宋体&quot;;</string>
       </property>
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_register_name">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>110</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_register_pw">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>170</y>
         <width>271</width>
         <height>20</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_register">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>300</y>
         <width>251</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_register_Rpw">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>240</y>
         <width>271</width>
         <height>21</height>
        </rect>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>210</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 12pt &quot;新宋体&quot;;</string>
       </property>
       <property name="text">
        <string>确认密码</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_re_return">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>21</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
